.changeset/dirty-carrots-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core': minor
+---
+
+Add className to the SidebarItem
```

Comment:
Doesn't seem to be a breaking change so should be `patch` instead of minor.

I am having difficulty in understanding the usecase of this change. Could you add a test for examples? Thanks!

Reply:
I added a screenshot of an example usage in the PR description